pkgs/by-name/bu/butler/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  pkgs,
```

Comment:
Include all dependencies directly instead of using pkgs

Reply:
I believe I've done so properly, I'm still very new to nix so it might need another look, sorry!